tests/unit/johnson_alahi_li_2016/test_modules.py

```diff
@@ -308,22 +281,46 @@ def test_transformer_smoke(subtests, image_medium):
         assert image_medium.size() == output_image.size()
 
 
-@pytest.mark.large_download
```

Comment:
The download was performed twice before: here and within the hub tests. Now we only check if `select_url` is called correctly and the correct `state_dict` is loaded here. The download stays the same in the hub tests.